### ISSUE TITLE
Install fails to create minikube VM

### ISSUE THREAD

**Issue by imorti** [kubernetes/minikube#259]
All-

Fresh clone and ran ```make ./out/minikube```. Then ```./out/minikube start``` and received this error. 

Installing just the binary gives the same error. Seems like we're just not creating the VM. 
Running: MAC OSX 10.11.5, Virtualbox Version 5.0.24 r108355

E0702 18:08:44.828290    6432 start.go:68] Error starting host:  Error getting state for host: machine does not exist

**Comment by dlorenc** [comment]
Can you try a `minikube delete`, followed by a `minikube start --show-libmachine-logs`, and attach the full output?

**Comment by imorti** [comment]
```minikube delete
Deleting local Kubernetes cluster...
Machine deleted.
Daniels-MBP:minikube imorti$ minikube start
Starting local Kubernetes cluster...
^C
Daniels-MBP:minikube imorti$ minikube delete
Deleting local Kubernetes cluster...
Machine deleted.
Daniels-MBP:minikube imorti$ minikube start --show-libmachine-logs
Starting local Kubernetes cluster...
Running pre-create checks...
Creating machine...
(minikubeVM) Downloading /Users/imorti/.minikube/cache/boot2docker.iso from https://storage.googleapis.com/minikube/minikube-0.4.iso...
(minikubeVM) 0%....10%....20%....30%....40%....50%....60%....70%....80%....90%....100%
(minikubeVM) Creating VirtualBox VM...
(minikubeVM) Creating SSH key...
(minikubeVM) Starting the VM...
(minikubeVM) Check network to re-create if needed...
(minikubeVM) Waiting for an IP...
Waiting for machine to be running, this may take a few minutes...
Detecting operating system of created instance...
Waiting for SSH to be available...
Detecting the provisioner...
Provisioning with boot2docker...
Copying certs to the local machine directory...
Copying certs to the remote machine...
Setting Docker configuration on the remote daemon...
Checking connection to Docker...
Docker is up and running!
Kubernetes is available at https://192.168.99.100:443.
Kubectl is now configured to use the cluster.```

Seems good now.
